index.ts

```diff
@@ -35,6 +37,19 @@ server.use(requestHandler.elevateRequest);
 
 // Route
 server.post('/request/:ns/:svName/:reqName', requestHandler.onRequest);
+server.post('/subscribe', util.rejectHTTP,
+                          apiSession.createSession,
+                          apiSession.handleSession,
+                          requestHandler.onSubscribe);
+server.get('/poll', util.rejectHTTP,
+                    apiSession.handleSession,
+                    requestHandler.onPoll);
+server.get('/unsubscribe', util.rejectHTTP,
```

Comment:
Let's remove the HTTP GET `/unsubscribe` logic since GET's should theoretically by idempotent (i.e., not change the server-side state).